exporters/dsexporter.go

```diff
@@ -107,6 +109,10 @@ func GetDataSourceAvailability(datasources []string, dsToCheck string) float64 {
 // Using a separate pedantic registry ensures that only your custom metric is exposed on the "/metrics" endpoint,
 // providing a cleaner and more focused view. Also reduces potential noise from unrelated metrics.
 func main() {
+	var bind string
+	flag.StringVar(&bind, "bind", ":8090", "The socket to bind to.")
```

Comment:
The value that we're providing to `bind` on the yaml is a combination of the address and the port, but the default value we give here contains only the port. Does it make sense?

Reply:
I removed the bind and the args from yaml. I think the args are not being used since we are directly using in the port to be 8090. Also, I checked that its being authenticated. No issues in removing the args.